### PR TITLE
Write tests for `compress_and_encrypt` util method.

### PR DESCRIPTION
**Pull request by mushtaqak**
**JIRA:** [ENT-1273](https://openedx.atlassian.net/browse/ENT-1273)

This PR adds the unit test for `compress_and_encrypt` util method. This will be followed by adding more tests for `pgpy` encryption work that @irfanuddinahmad is working.

**Related PR:** https://github.com/edx/edx-enterprise-data/pull/73

### CODE REVIEW

File: enterprise_reporting/tests/test_utils.py [hunk 1]
```diff
@@ -147,3 +149,73 @@ def test_flatten_dict_weird_cases(self, dictionary):
...
+        zipfile = ZipFile(compressed_file, 'r')
+
+        for file_data in files_data:
+            # Verify text file is present in zip file.
+            self.assertIn(file_data['name'], zipfile.namelist())
+
+            # Verify file content is readable is correct password.
+            content = zipfile.read(file_data['name'], password)
+            self.assertIsNotNone(content)
+
```
Comment (irfanuddinahmad) [inline enterprise_reporting/tests/test_utils.py:224]: You could compare the size of content to be equal to the original data created in the file

File: enterprise_reporting/tests/test_utils.py [hunk 2]
```diff
@@ -147,3 +149,73 @@ def test_flatten_dict_weird_cases(self, dictionary):
...
+            # Also verify file is only accessible with correct password.
+            with self.assertRaises(RuntimeError):
+                zipfile.read(file_data['name'], 'gollum')
+
+            # Remove text file now.
+            os.remove(file_data['name'])
+
+        # Remove compressed file now.
+        os.remove(compressed_file)
```
Comment (irfanuddinahmad) [inline enterprise_reporting/tests/test_utils.py]: nit: line at the end

Comment (irfanuddinahmad) [inline enterprise_reporting/tests/test_utils.py]: You could look at https://docs.python.org/2/library/tempfile.html

**Comment by mushtaqak** [comment]
@irfanuddinahmad @zubair-arbi ready for review again

**Review by zubair-arbi — APPROVED** [review]
LGTM 👍

**Comment by saleem-latif** [comment]
@irfanuddinahmad Tests are now passing, it is ready for review.

**Review by irfanuddinahmad — APPROVED** [review]
LGTM

**Pull request merged**